configure.ac

```diff
@@ -20,19 +22,39 @@ AC_HEADER_STDC
 # Check endianness
 AC_MSG_CHECKING([system endianness])
 AC_PREPROC_IFELSE(
-  [AC_LANG_PROGRAM([[#include <endian.h>]],
-		   [[#if __BYTE_ORDER != __LITTLE_ENDIAN]]
-		   [[#error]]
-		   [[#endif]])],
+  [AC_LANG_PROGRAM([[#if !_AIX]]
+            [[#include <endian.h>]]
+            [[#endif]],
+            [[#if !_AIX]]
+		    [[#if __BYTE_ORDER != __LITTLE_ENDIAN]]
+		    [[#error]]
+		    [[#endif]]
+            [[#endif]])],
```

Comment:
This has several problems.  First, I don't' think it's necessary to have the two sequential conditionals like this; that is, they could both be under a single #if !_AIX - #endif.  I my testing it gave the same result when done that way: i.e. just remove the inner-most `[[#endif]]` and `[[#if !_AIX]]`.  (not just the same result overall, but the same result per test).

But really, with that conditional added, it produces a test that will always be a no-op on AIX.  I'd prefer to find a way skip these tests on AIX rather than generating and testing useless code.